VRTweaks/VRTweaks.csproj

```diff
@@ -100,14 +100,17 @@
   <ItemGroup>
     <Compile Include="Main.cs" />
     <Compile Include="Properties\AssemblyInfo.cs" />
-    <Compile Include="GazeCursor.cs" />
     <Compile Include="SnapTurn\Config.cs" />
     <Compile Include="SnapTurn\SnapTurning.cs" />
+    <Compile Include="HUDFixer.cs" />
   </ItemGroup>
   <ItemGroup>
     <None Include="mod.json">
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </None>
   </ItemGroup>
   <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
+  <PropertyGroup>
+    <PostBuildEvent>copy "$(ProjectDir)bin\Debug\VRTweaks.dll" "$(CommonDir)SubnauticaZero\QMods\VRTweaks"</PostBuildEvent>
```

Comment:
This was a handy thing I added so you don't have to manually copy the updated VRTweaks.dll to the BZ directory each time you test a change. Can do the same thing for the other project in this solution